### PR TITLE
 Include SCA policy for Rocky Linux 9 in SPECS 

### LINKED ISSUE #2282: SPECS for new SCA Rocky Linux 9
Create SPECS as needed for a new OS SCA

PR - https://github.com/wazuh/wazuh/pull/17810 Is ready to merge
Branch - https://github.com/wazuh/wazuh/tree/SCA-Rocky-Linux-9

PLEASE confirm with @wazuh/threat-intel before merging.

### PR DESCRIPTION
**Pull request by DFolchA**
|Related issue|
|---|
|Closes #2282  |

## Description

This PR adds the necessary modifications to the SPECS to include the new Rocky Linux 9 SCA policy.

## Tests

- Build the package in any supported platform
  - [x] Linux
  - [ ] Windows
  - [ ] macOS
  - [ ] Solaris
  - [ ] AIX
  - [ ] HP-UX
- [x] Package installation
- [x] Package upgrade
- [x] Package downgrade
- [x] Package remove
- [x] Package install/remove/install
- [ ] Change added to CHANGELOG.md

- Tests for Linux RPM
  - [x] Build the package for x86_64
  - [ ] Build the package for i386
  - [ ] Build the package for armhf
  - [ ] Build the package for aarch64
  - [ ] `%files` section is correctly updated if necessary
- Tests for Linux deb
  - [x] Build the package for x86_64
  - [ ] Build the package for i386
  - [ ] Build the package for armhf
  - [ ] Build the package for aarch64
  - [ ] Package install/remove/install
  - [ ] Package install/purge/install
  - [ ] Check file permissions after installing the package
- Tests for macOS
  - [ ] Test the package from macOS Sierra to Mojave
- Tests for Solaris
  - [ ] Test the package on Solaris 10
  - [ ] Test the package on Solaris 11
  - [ ] Check file permissions on Solaris 11 template
- Tests for IBM AIX
  - [ ] `%files` section is correctly updated if necessary
  - [ ] Check the changes from IBM AIX 5 to 7

### CODE REVIEW

**Review by vikman90 — DISMISSED** [review]

**Review by ncvicchi — DISMISSED** [review]

File: rpms/SPECS/wazuh-manager.spec [hunk 1]
```diff
@@ -338,10 +340,6 @@ if [ $1 = 1 ]; then
   %{_localstatedir}/packages_files/manager_installation_scripts/add_localfiles.sh %{_localstatedir} >> %{_localstatedir}/etc/ossec.conf
 fi
```
Comment (lchico) [inline rpms/SPECS/wazuh-manager.spec:342]: Why we are removing this:
```

if [[ -d /run/systemd/system ]]; then
  rm -f %{_initrddir}/wazuh-manager
fi
```

**Review by lchico — CHANGES_REQUESTED** [review]
Thanks, @mjcr99. I left one comment, PTAL.

File: rpms/SPECS/wazuh-manager.spec (hunk 1 above)
Comment (mjcr99) [reply rpms/SPECS/wazuh-manager.spec]: Fixed in https://github.com/wazuh/wazuh-packages/pull/2299/commits/c78f49f962d59bee37ef0c2f165b150cb4371f4b well spotted @lchico, thanks!

File: rpms/SPECS/wazuh-agent.spec [hunk 2]
```diff
@@ -334,7 +335,6 @@ SCA_BASE_DIR="%{_localstatedir}/tmp/sca-%{version}-%{release}-tmp"
 mkdir -p %{_localstatedir}/ruleset/sca
 
 SCA_TMP_DIR="${SCA_BASE_DIR}/${SCA_DIR}"
-
```
Comment (lchico) [inline rpms/SPECS/wazuh-agent.spec:337]: NIS: We shouldn't remove this blank line.

**Review by lchico — APPROVED** [review]
I left one minor comment! Feel free to address this.

File: rpms/SPECS/wazuh-agent.spec (hunk 2 above)
Comment (mjcr99) [reply rpms/SPECS/wazuh-agent.spec:337]: Addressed https://github.com/wazuh/wazuh-packages/pull/2299/commits/017d620c5aed9a8231c49bed65e78dc9702adadd

**Review by jotacarma90 — APPROVED** [review]
LGTM

**Pull request merged**